CHANGELOG

```diff
@@ -1,3 +1,9 @@
+2.0.3
+=====
+FIXES:
+* Update to Ruby 2.5.0
+* Update bundled Gems
```

Comment:
`.ruby-version` as well as `Gemfile.lock` are not packaged in the gem so I don't think we need to mention this in the CHANGELOG (i.e. it only applies to developers of the gem, not end-users)

Reply:
@toupeira 

But as far as I understood, I need to lift the version number tho. Right?
So shal I not mention any of the changes in the Changelog then?

Reply:
IMHO lifting the version number is not necessary either because none of the changes apply to the end-users of the gem (same reasoning for not mentioning it in the CHANGELOG). What do you think @huerlisi?

And usually with gem PRs it's better to not touch the version at all and let the maintainers handle it after merging, because you might want to combine several PRs for a new release.

Reply:
@toupeira who is the maintainers? 😉 Do the maintainers have spare time to care  😜

Reply:
@toupeira but as it doesn't affect gem end users functionality, you will probably be right about not lifting it.

Reply:
@casapar that would be @huerlisi, but I just noticed you have commit access so I guess you're a maintainer too ;-)

Reply:
@casaper has been promoted to be a maintainer some time ago:-) Big thanks for living up to this role!